backend/benefit/applications/services/ahjo_integration.py

```diff
@@ -13,23 +14,26 @@
 TEMPLATE_ID_BENEFIT_WITHOUT_DE_MINIMIS_AID = "benefit_without_de_minimis_aid"
 TEMPLATE_ID_BENEFIT_DECLINED = "benefit_declined"
 TEMPLATE_ID_COMPOSED_ACCEPTED_PUBLIC = "composed_accepted_public"
+TEMPLATE_ID_COMPOSED_DECLINED_PUBLIC = "composed_declined_public"
 TEMPLATE_ID_COMPOSED_ACCEPTED_PRIVATE = "composed_accepted_private"
 TEMPLATE_ID_COMPOSED_DECLINED_PRIVATE = "composed_declined_private"
 
 JINJA_TEMPLATES_COMPOSED = {
     TEMPLATE_ID_COMPOSED_ACCEPTED_PUBLIC: {
         "path": "composed_accepted_public.html",
-        "file_name": "Liite 1 Helsinki-lisä 2021 koontiliite yhteisöille "
-        "julkinen.pdf",
+        "file_name": "Liite 1 Helsinki-lisä koontiliite julkinen.pdf",
+    },
+    TEMPLATE_ID_COMPOSED_DECLINED_PUBLIC: {
+        "path": "composed_rejected_public.html",
+        "file_name": "Liite 1 Helsinki-lisä koontiliite julkinen.pdf",
     },
     TEMPLATE_ID_COMPOSED_ACCEPTED_PRIVATE: {
         "path": "composed_accepted_private.html",
-        "file_name": "Helsinki-lisä 2021 koontiliite yhteisöille "
-        "salassa pidettävä.pdf",
+        "file_name": "Helsinki-lisä koontiliite salassa pidettävä.pdf",
     },
     TEMPLATE_ID_COMPOSED_DECLINED_PRIVATE: {
         "path": "benefit_declined.html",  # Composed decline template reuse single company decline template
-        "file_name": "Helsinki-lisä 2021 kielteiset päätökset koontiliite "
+        "file_name": "Helsinki-lisä kielteiset päätökset koontiliite "
         "yrityksille salassa pidettävä.pdf",
     },
 }
```

Comment:
The file_name values are inconsistent and they're not unique because the accepted public filename is the same as the declined public filename.  Also the decline private has "yrityksille" part in it but none of the others don't although they previously had "yhteisöille" in them. Please could you check these filenames out and update them, thank you!

Reply:
fixed